### DISCUSSION TITLE
Document the `nuxi prepare` command

### DISCUSSION THREAD

**Discussion by Lyokolux** [nuxt/nuxt#16927] [Ideas]
I have found a use case for this command:
* [Histoire](https://histoire.dev/) needs the generated `tsconfig` of Nuxt.

In this cases, `nuxi prepare` avoids the build of the complete app. It is then efficient in CI.

A `prepare` command with `yarn` or `npm` could be useful. This `prepare` command will run after the installation of the packages and make the nuxt app ready for future usage.

I think it is then relevant to include it in the documentation of Nuxt 3 next to https://v3.nuxtjs.org/api/commands/preview.

What do you think?

**Comment by huang-julien** [comment]
I also use it before running tests in CI :+1:
